src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1471,6 +1471,29 @@ class C
             End Using
         End Function
 
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        <WorkItem(13527, "https://github.com/dotnet/roslyn/issues/13527")>
+        Public Async Function TestImplicitObjectCreateExpression() As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+                  <Document><![CDATA[
+public class C
+{
+    public C(int Alice, int Bob) { }
+    public C(string ignored) { }
+
+    public void M()
+    {
+        C c = new($$)
+    }
+}]]></Document>, languageVersion:=LanguageVersion.Preview)
+
+                state.SendTypeChars("A")
+                Await state.AssertSelectedCompletionItem(displayText:="Alice:", isHardSelected:=True)
+                state.SendTypeChars(":")
+                Assert.Contains("new(Alice:", state.GetLineTextFromCaretPosition(), StringComparison.Ordinal)
+            End Using
+        End Function
```

Comment:
what about completoin after you type `C c = new(`.  We likely have an open design problem of "do we select/complete the type, or not?"  We may even need an option for this!! <gasp> :D

Reply:
Should probably open an issue for that that we can tlak about in design meeting.  you can come! :)

Reply:
I didn't understand. There is not type to complete after you've typed an open-paren (tuples are not allowed in object creation syntax)

Reply:
Discussed offline.